designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala

```diff
@@ -53,7 +53,7 @@ class SettingsResources(config: FeatureTogglesConfig,
 
 @JsonCodec case class RemoteEnvironmentConfig(targetEnvironmentId: String)
 
-@JsonCodec case class EnvironmentAlert(content: String, cssClass: String)
+@JsonCodec case class EnvironmentAlert(content: String, color: String)
```

Comment:
I think we should add info about this in the changelog, it's incompatible change in case of the configuration